.github/workflows/on-push.yml

```diff
@@ -23,35 +20,107 @@ jobs:
       - run: yarn run lint
       - run: yarn run build:release
 
-  test:
-    runs-on: ubuntu-latest
+  unit-test:
+    runs-on: ubuntu-22.04
 
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 14
-      - run: docker-compose up -d app
       - uses: volta-cli/action@v4
       - uses: actions/cache@v2
         with:
           path: '**/node_modules'
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
       - run: yarn install --frozen-lockfile
+      - name: create .env file
+        run: |
+          echo "${{ secrets.E2E_GITHUB_PRIVATE_KEY }}" > jira-test.pem
+          echo "APP_URL=http://localhost" > .env
+          echo "WEBHOOK_PROXY_URL=http://localhost" >> .env
+          echo "INSTANCE_NAME=fusion-test" >> .env
+          echo "NODE_ENV=test" >> .env
+          echo "APP_NAME=jira" >> .env
+          echo "APP_ID=${{ secrets.E2E_GITHUB_APP_ID }}" >> .env
+          echo "WEBHOOK_SECRET=${{ secrets.E2E_GITHUB_WEBHOOK_SECRET }}" >> .env
```

Comment:
The step is called `unit-test`, but the secret is called `E2E_xxxx`, seems a bit strange to me. Am I missing something?

Reply:
nah, just copy pasted it from the e2e tests.  Not sure this is even needed as unit tests never go out to the internet.  I'll review this